references/obj_detection/train_pytorch.py

```diff
@@ -21,32 +22,57 @@
 
 from doctr.datasets import DocArtefacts
 from doctr.utils import DetectionMetric
+from doctr.transforms.functional.pytorch import rotate
 
 
-def convert_to_abs_coords(targets, img_shape):
+def convert_to_abs_coords(targets, img_shape, is_transform=False, **kwargs):
     height, width = img_shape[-2:]
-
     for idx in range(len(targets)):
         targets[idx]['boxes'][:, 0::2] = (targets[idx]['boxes'][:, 0::2] * width).round()
         targets[idx]['boxes'][:, 1::2] = (targets[idx]['boxes'][:, 1::2] * height).round()
-
-    targets = [{
-        "boxes": torch.from_numpy(t['boxes']).to(dtype=torch.float32),
-        "labels": torch.tensor(t['labels']).to(dtype=torch.long)}
-        for t in targets
-    ]
-
-    return targets
+    if is_transform:
+        if bool(random.getrandbits(1)):
+            angle = random.uniform(0.1, 5.0)
+        else:
+            angle = random.uniform(-0.1, -5.0)
```

Comment:
you won't want the same angle for the entire batch, better to do this as follows with a `max_angle`
```python
angles = 2 * max_angle * (np.random.rand(len(targets)) - 1)
```
and then apply each angle to each image set of targets